tests/test_mcp_servers.py

```diff
@@ -0,0 +1,107 @@
+# Code bellow written following examples here: https://llama-stack.readthedocs.io/en/latest/building_applications
+from llama_stack_client.lib.agents.agent import Agent
+from llama_stack_client.lib.agents.event_logger import EventLogger
+from llama_stack_client import LlamaStackClient
+import argparse
+import logging
+import os
+from dotenv import load_dotenv
+import json
+
+
+
+
+def test_mcp_server(file_path,REMOTE_MCP_URL,model, mcp_server_name):
```

Comment:
total nit, sorry 😃 : 
```suggestion
def test_mcp_server(file_path, REMOTE_MCP_URL, model, mcp_server_name):
```